client/src/components/GameImage/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import GameFocused from '../GameFocused';
+
+const Item = styled.li`
+  margin-bottom: 0;
+  margin-bottom: ${p => (p.selected === true ? '440px' : '0')};
+  transition: all ease-in ${p => p.transitionSpeed};
+`;
+
+const ImgWrapper = styled.div`
+  width: auto;
+  position: relative;
+  border: 3px solid;
+  border-radius: 2px;
+  box-sizing: border-box;
+  border-color: ${p =>
+    p.selected === true ? p => p.theme.color.white : 'transparent'};
+
+  &:hover {
+    border-color: ${p => p.theme.color.white};
+  }
+  &:after {
+    bottom: ${p => (p.selected === true ? '-20px' : '0')};
+    opacity: ${p => (p.selected === true ? '1' : '0')};
+    content: '';
+    width: 0;
+    height: 0;
+    position: absolute;
+    left: calc(50% - 20px);
+    border-radius: 2px;
+    border-left: 20px solid transparent;
+    border-right: 20px solid transparent;
+    border-top: 20px solid ${p => p.theme.color.white};
+    transition: bottom 0.3s ease-in-out;
+    z-index: -1;
+  }
+`;
+
+const GameImg = styled.img`
+  max-width: 100%;
+  max-height: 100%;
+  height: 260px;
+`;
+
+const GameImage = ({
+  name,
+  image,
+  id,
+  handleSelected,
+  transitionSpeed,
+  selected,
+  ...props
+}) => {
+  return (
+    <Item
+      onClick={e => handleSelected(e.currentTarget, id)}
+      selected={selected}
+      transitionSpeed={transitionSpeed}
+    >
+      <ImgWrapper selected={selected}>
+        <GameImg src={image} title={name} alt={name} />
+      </ImgWrapper>
+      {selected && <GameFocused {...props} name={name} image={image} />}
+    </Item>
+  );
+};
+
+GameImage.prototypes = {
```

Comment:
I think you editor has been auto filling `prototypes`. This is supposed to be `propTypes`
```suggestion
GameImage.propTypes = {
```